docs/docs/mailer.md

```diff
@@ -0,0 +1,223 @@
+# Mailer
+
+Redwood provides a convenient-to-use mailer that you can reach for when you need to start sending emails to your users.
+
```

Comment:
Perhaps add an overview section here with benefits and goals.
For example, how is the sanding great- how is testing unique.

Play up the feature.

Reply:
For example: Need a safe way that emails don’t send in wild in development accidentally and in test